pkgs/top-level/all-packages.nix

```diff
@@ -3006,6 +3006,8 @@ let
 
   super = callPackage ../tools/security/super { };
 
+  superUserSpark = haskellPackages.callPackage ../applications/misc/super_user_spark { };
```

Comment:
Please rename to `super-user-spark`. We prefer to use dashes in the attribute names now too :)